handlers/facebookapp/facebookapp.go

```diff
@@ -56,18 +56,20 @@ const (
 	payloadKey    = "payload"
 )
 
+func newHandler(channelType courier.ChannelType, name string, validateSignatures bool) courier.ChannelHandler {
```

Comment:
`validateSignatures` is misleading as that parameter is for using channel UUIDS routes or not